vhdbuilder/parts/k8s/cloud-init/artifacts/kubelet.service

```diff
@@ -36,5 +36,7 @@ ExecStart=/usr/local/bin/kubelet \
         $KUBELET_CONFIG \
         $KUBELET_REGISTER_NODE $KUBELET_REGISTER_WITH_TAINTS
 
+Slice=kubesystem.slice
```

Comment:
@xuto2 @adilliadil I'm really struggling to get this change to propagate into the final image /etc/systemd/system/kubelet.service.

I've run several builds off the last few commits, and all the other changes end up working in the final image. For some reason this doesn't. Any ideas? Example: https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=30772758&view=results -> builds SIG image 1.0.2, which I built a cluster with and did not find the changes in the image.

Reply:
Is this file coming from some external source?